home-manager-experimental.nix

```diff
@@ -6,12 +6,14 @@ let
   caches =
     let toCachix = import ./fetchCachix.nix;
     in map toCachix cfg.cachix ++ cfg.extraCaches;
+
   substituters = concatStringsSep " " (map (v: v.url) caches);
   publicKeys = concatStringsSep " " (concatMap (v: v.keys or [ v.key ]) caches);
-  nixConfSource = ''
-    extra-substituters = ${substituters}
-    extra-trusted-public-keys = ${publicKeys}
-  '';
+
+  nixSettings = {
+    extra-substituters = substituters;
+    extra-trusted-public-keys = publicKey;
```

Comment:
Typo here, `publicKey` ➡️ `publicKeys`

Reply:
Thanks! fixed in abe4d70e5d9a225a7ecf98086b089cc04202f2e3.